p8_integration_tests/scripts_test/build_script.py

```diff
@@ -25,13 +27,14 @@ def add_scripts(a_dir, prefix_len, test_file, exceptions, broken):
         if os.path.isdir(script_path) and not a_script.startswith("."):
             add_scripts(script_path, prefix_len, test_file, exceptions, broken)
         if a_script.endswith(".py"):
-            print(script_path)
             name = script_path[prefix_len:-3].replace(os.sep, "_")
-            print(name)
             test_file.write("\n    def ")
             test_file.write(name)
             test_file.write("(self):\n        self.check_script(\"")
-            test_file.write(os.path.abspath(script_path))
+            the_path = os.path.abspath(script_path)
+            if platform.system() == "Windows":
+                the_path= the_path.replace("\\","/")
```

Comment:
If you only ever use the os.path functions (join etc.), you shouldn't need to do this.  Not sure if you have experienced a problem though.

Reply:
I have experienced a problem. Windows ONLY.
The issue is that we go from the output of path to a string written in a file and then back to a path.

The \\ is seen in the string as a single \ which is read as make the next character special and not as the path separator.

Reply:
OK